pkg/engine/artifacts.go

```diff
@@ -13,258 +13,258 @@ import (
 	"github.com/Azure/aks-engine/pkg/api/common"
 )
 
-type kubernetesFeatureSetting struct {
-	sourceFile      string
-	destinationFile string
-	isEnabled       bool
-	rawScript       string
+type kubernetesComponentFileSpec struct {
+	sourceFile      string // filename to source spec data from
+	base64Data      string // if not "", this base64-encoded string will take precedent over sourceFile
```

Comment:
renamed `rawScript` to `base64Data` to more accurately express what the value is (arbitrary base64-encoded string data)